docs/MainSite/Installation.md

```diff
@@ -47,45 +47,46 @@ See LICENSE.txt for details.
 A [Docker](https://www.docker.com/) image is available from
 [DockerHub](https://hub.docker.com/r/sxscollaboration/spectrebuildenv/) and can
 be used to build SpECTRE on a personal machine.
-
-**Note**: The Docker image is the recommended way of using SpECTRE on a personal
-Linux machine. Because of the wide variety of operating systems available today
+This is the recommended way of using SpECTRE on a personal
+Linux or MacOS machine. (Because of the wide variety of operating systems available today
 it is not possible for us to support all configurations. However, using Spack
-as outlined below is a supported alternative to Docker images.
+as outlined below is a supported alternative to Docker images.)
 
 To build with the docker image:
 
-1. Clone SpECTRE into SPECTRE_ROOT, a directory of your choice.
-2. Retrieve the docker image (you may need `sudo` in front of this command)
+1. Clone SpECTRE into a directory of your choice.
+1. `export SPECTRE_ROOT=/path/to/your/spectre`
```

Comment:
Maybe instruct users to add this to their shell environment so they don't have to run it every time they log in?

Reply:
Also fix bullet point numbering

Reply:
This is a correct way to enumerate in markdown.  It is rendered correctly

Reply:
So it is, good to know!